### PR TITLE
fix: separate test attempts

### PR DESCRIPTION
**Pull request by limpbrains**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/ggro6kilssje76ptkc2odmyv5i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11616)

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -88,18 +93,54 @@ jobs:
...
+    - name: Test attempt 4
+      uses: reactivecircus/android-emulator-runner@v2
+      if: steps.test3.outcome != 'success'
+      with:
+        api-level: 31
+        avd-name: Pixel_API_29_AOSP
+        force-avd-creation: false
+        emulator-options: -no-window -gpu swiftshader_indirect -no-snapshot -noaudio -no-boot-anim -camera-back none -camera-front none -partition-size 2047
+        arch: x86_64
+        script: npm run e2e:release-test
```
Comment (Overtorment) [inline .github/workflows/ci.yml:143]: whats the benefit over just

`npm run e2e:release-test || npm run e2e:release-test || npm run e2e:release-test || npm run e2e:release-test`

?

**Comment by limpbrains** [comment]
Log output is much easier to read, take a look

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Comment by Overtorment** [comment]
oops it got merged

**Pull request merged**